gtkmm-plplot/plot2d.cpp

```diff
@@ -204,6 +209,28 @@ bool Plot2D::get_axis_logarithmic_y() {
   return log10_y;
 }
 
+void Plot2D::set_axis_time_format(Glib::ustring _time_format)
+{
+  if (!_time_format.empty())
+  {
+    if (get_axis_logarithmic_x())
+      throw Exception("Gtkmm::Plplot::Plot2D::set_axis_time_format -> X-axys cannot be time and logarithmic");
```

Comment:
please replace axys with axis.

Reply:
also please follow my convention for positioning curly braces

Reply:
Are you not planning to support datetimes on the Y-axis as well? You should rename `Plot2D::set_axis_time_format` to `Plot2D::set_axis_time_format_x` and `Plot2D::get_axis_time_format` to `Plot2D::get_axis_time_format_x`.